.github/dependabot.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+updates:
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    groups:
+      actions:
+        patterns:
+          - "*"
```

Comment:
This patterns will run the Dependabot workflow on a weekly schedule and will group all updates it finds into 1 PR to avoid noise from multiple updates.